notebooks/en/function_calling_fine_tuning_llms_on_xlam.ipynb

```diff
@@ -0,0 +1,14340 @@
+{
```

Comment:
<p>This blocks feel ok for a <code>.py</code> script but too much for a learning  notebook. It could benefit from dividing these type of blocks intro smaller blocks, with explanations</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>